cmd2/ansi.py

```diff
@@ -193,8 +198,13 @@ def style(text: Any, *, fg: str = '', bg: str = '', bold: bool = False,
 # These can be altered to suit an application's needs and only need to be a
 # function with the following structure: func(str) -> str
 style_success = functools.partial(style, fg='green')
+"""Partial function supplying arguments to :meth:`cmd2.ansi.style()` to generate bold green text"""
```

Comment:
I didn't know about the "#: " comment trick for Sphinx. So I did some digging, and it turns out that both the comment approach and the bare string approach work in Sphinx. From https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html:

> For module data members and class attributes, documentation can either be put into a comment with special formatting (using a #: to start the comment instead of just #), or in a docstring after the definition. Comments need to be either on a line of their own before the definition, or immediately after the assignment on the same line. The latter form is restricted to one line only.

Note that this only works for module data members (which is what `style_success` is) and class attributes.

Reply:
Should we pick one or the other (I don't care which) and specify it in our doc_conventions?

Reply:
I think the way you are doing it allows for multiline comments whereas the one I was suggesting only allows single-line comments.  At least some of the time multiline comments would be highly desirable so I’d say let’s go with your way.

Reply:
I've added this to `doc_conventions.rst`